frontend/src/main/components/CourseDetails/CourseDetailsTable.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import OurTable from "main/components/OurTable";
+import {
+  convertToFraction,
+  formatInstructors,
+  formatLocation,
+  formatTime,
+} from "main/utils/sectionUtils.js";
+
+export default function PersonalSectionsTable({ details }) {
```

Comment:
At a minimum, this should be changed to match the name of the file

```suggestion
export default function CourseDetailsTable({ details }) {
```

Reply:
But if you take that approach, at a minimum, create a separate refactoring issue that consolidates PersonalSectionsTable and CourseDetailsTable into a single component OR ELSE explains clearly in comments how they are different (other than name and where they are used.)

That's the minimum I'd ask to get this mergeable.  But it would be better to use the existing component.